content/docs/command-reference/config.md

```diff
@@ -9,7 +9,8 @@ usage: dvc config [-h] [--global | --system | --local] [-q | -v] [-u]
                   [-l] [name] [value]
 
 positional arguments:
-  name           Option name (section.option or remote.name.option).
+  name     Option name in format: section.option or remote.name.option
```

Comment:
something wrong with spaces (or tabs?) here?

Reply:
I removed spacing in my suggestion so that code block doesn't scroll horizontally.

Reply:
👍